### ISSUE TITLE
Exceptions.

### ISSUE THREAD

**Issue by 89gsc** [mehedimi/wp-query-builder#5]
keep getting fatal errors which are not being prevented by wrapping code in try {} statements to catch exceptions. 

Please post example usage of try / catch with query exceptions etc if possible. 

Error Example: 

```NOTICE: PHP message: PHP Warning:  mysqli_stmt::bind_param(): Number of variables doesn't match number of parameters in prepared statement in /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Connection.php on line 193
NOTICE: PHP message: PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Mehedi\WPQueryBuilder\Connection::getRowsFromResult() must be an instance of mysqli_result, bool given, called in /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Connection.php on line 82 and defined in /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Connection.php:92
Stack trace:
#0 /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Connection.php(82): Mehedi\WPQueryBuilder\Connection->getRowsFromResult()
#1 [internal function]: Mehedi\WPQueryBuilder\Connection->Mehedi\WPQueryBuilder\{closure}()
#2 /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Connection.php(56): call_user_func()
#3 /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Connection.php(83): Mehedi\WPQueryBuilder\Connection->run()
#4 /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Query/Builder.php(220): Mehedi\WPQueryBuil in /app/web/app/mu-plugins/gc-api-routes/vendor/mehedimi/wp-query-builder/src/Connection.php on line 92
192.168.112.224 -  02/Dec/2022:15:31:44 +0000 "GET /index.php" 500```

**Comment by mehedimi** [comment]
Thanks for creating this issue. I will fix this issue ASAP.

**Comment by mehedimi** [comment]
I have released version `0.8`. I hope this will fix your issue. Please update your package by running `composer update`
